test/mysql-chassis.js

```diff
@@ -316,6 +326,10 @@ describe('mysql-chassis', () => {
 
   // })
 
+  // describe('transaction methods', () => {
+
```

Comment:
Are you planning on updating the PR with these tests later? Just want to confirm, thanks.